bot-config.json

```diff
@@ -1,5 +1,5 @@
 {
     "discord": {
-        "token": "<Njc1ODk5MzI1NTU3MjQzOTM1.Xj929g.driQBVFfZZL-MlthnY0XtiSr4Lo>"
+        "token": "NzAzODkzMzU0Mzk5OTI0MjY1.XqVO5g.bZ_jPCb7O_bMFQxrPy08d5G1uNs"
```

Comment:
We'll probably want to remove the token here.

Reply:
It's already done automatically by discord when you upload the token online.
So that represents no security issue 'cuz you need a new token in order to successfully run the bot instance. 
I'll be more careful on my next pull request.